measure_extinction/extdata.py

```diff
@@ -235,6 +234,7 @@ def __init__(self, filename=None):
         self.waves = {}
         self.exts = {}
         self.uncs = {}
+        self.stds = {}
```

Comment:
I'm not sure that this should be defined by default?  Won't this mean the write function will always try and write this column even if it hasn't been calculated?

Reply:
Are you referring to the save function? The save function has
`if curname in self.stds.keys()` and should thus only write this column if the stds has been calculated for that part of the curve...

Reply:
If I do not define `self.stds` as an empty library, I get an AttributeError: 'ExtData' object has no attribute 'stds' when trying to add the standard deviation...

Reply:
But isn't the stds part of the keys() even if it is an empty dictonary?
When you calculate stds in the averaging process, define it there before you use it.  You can add to the class variables anywhere you want.

Reply:
The keys() are e.g. IUE, BAND, etc... If I try to define stds in the averaging process, I get an error:
`aveext.stds = {}`
AttributeError: 'ExtData' object has no attribute 'stds'

Reply:
Good point.  I retract my comment.